code/modules/awaymissions/maploader/reader.dm

```diff
@@ -16,10 +16,22 @@ var/global/dmm_suite/preloader/_preloader = null
  * 2) Read the map line by line, parsing the result (using parse_grid)
  *
  */
-/dmm_suite/load_map(dmm_file as file, z_offset as num)
+
+/dmm_suite/load_map(dmm_file as file, x_offset as num, y_offset as num, z_offset as num)
+
 	if(!z_offset)//what z_level we are creating the map on
 		z_offset = world.maxz+1
 
+	if(!x_offset)

```

Comment:
define them IN the args as 0.
so x_offset = 0, y_offset = 0
none of this "as num" garbage.
also remove the "as num"s

dmm_file can probably be file/dmm_file instead of "as file"